settings.gradle

```diff
@@ -14,6 +14,13 @@ buildscript {
     }
 }
 
+buildCache {
+    local {
+        directory = new File(rootDir, '.build-cache')
+        removeUnusedEntriesAfterDays = 30
+    }
+}
```

Comment:
And you did see improvements in CI here too? Otherwise this seems not do much on a clean CI run. But I guess it also doesn't have downsides to have it on. 🤷

Reply:
I think so? It's hard to say due to noise in durations of the CI runs. Caches are certainly being used in CI, so it should help as long as we're not taking too long packing outputs for the cache.